content/docs/examples/advanced-gateways/ingress-certmgr/index.md

```diff
@@ -0,0 +1,174 @@
+    name: http
+  selector:
+    app: helloworld
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: helloworld
+spec:
+  template:
+    metadata:
+      labels:
+        app: helloworld
+    spec:
+      containers:
+      - name: helloworld
+        image: istio/examples-helloworld-v1
+        resources:
+          requests:
+            cpu: "100m"
+        imagePullPolicy: IfNotPresent
+        ports:
+        - containerPort: 5000
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  annotations:
+    kubernetes.io/ingress.class: istio
+  name: helloworld-ingress
+spec:
+  rules:
+    - host: "$INGRESS_DOMAIN"
+      http:
+        paths:
+          - path: /hello
+            backend:
+              serviceName: helloworld
+              servicePort: 5000
+---
+EOF
+{{< /text >}}
+
+{{< tip >}}
+Notice the use of `INGRESS_DOMAIN` variable we defined earlier
+{{< /tip >}}
+
+Now we should be able to access our demo application via HTTP:
+
+{{< text bash >}}
+$ curl http://$INGRESS_DOMAIN/hello
+Hello version: v1, instance: helloworld-5d498979b6-jp2mf
+{{< /text >}}
+
+HTTPS access still won't work as we don't have any TLS certificates. Let's fix that.
+
+## Getting a "Let's Encrypt" certificate issued using the "cert-manager"
+
+At this point our Istio installation should have "cert-manager" up and running with two `ClusterIssuer` resources configured (for production and staging ACME-endpoints provided by "Let's Encrypt"). We will be using staging endpoint for this tutorial (feel free to try swapping `letsencrypt-staging` for `letsencrypt` to get a browser-trusted certificate issued).
+
+In order to have a certificate issued and managed by the "cert-manager" we need to create a `Certificate` resource:
+
+{{< text bash >}}
+$ cat <<EOF | kubectl apply -f -
+apiVersion: certmanager.k8s.io/v1alpha1
+kind: Certificate
+metadata:
+  name: ingress-cert-staging
+  namespace: istio-system
+spec:
+  secretName: ingress-cert-staging
+  issuerRef:
+    name: letsencrypt-staging
+    kind: ClusterIssuer
+  commonName: $INGRESS_DOMAIN
+  dnsNames:
+  - $INGRESS_DOMAIN
+  acme:
+    config:
+    - http01:
+        ingressClass: istio
+      domains:
+      - $INGRESS_DOMAIN
+---
+EOF
+{{< /text >}}
+
+Notice that the `secretName` matches the `credentialName` attribute value that we previously used while configuring the `Gateway` resource. The `Certificate` resource will be processed by the "cert-manager" and a new certificate will eventually be issued. Consult the status of the `Certificate` resource to check the progress:
+
+{{< text bash >}}
+$ kubectl -n istio-system describe certificate ingress-cert-staging
+-> status should eventually flip to 'Certificate issued successfully'
+{{< /text >}}
+
+At this point the service should become available over HTTPS as well:
+
+{{< text bash >}}
+$ curl --insecure http://$INGRESS_DOMAIN/hello
+Hello version: v1, instance: helloworld-5d498979b6-jp2mf
+{{< /text >}}
```

Comment:
example with https?

Reply:
Should be "https" of course (and it has been tested that way), nicely spotted, thanks!